### ISSUE TITLE
Could not find a declaration file for module `@electron/remote/main`

### ISSUE THREAD

**Issue by cawa-93** [electron/remote#52]
I just install ` @electron/remote@1.1.0`. 
```ts
// main.ts
import {initialize} from '@electron/remote/main'; // TS7016: Could not find a declaration file for module '@electron/remote/main'.

initialize();
```

As ugly workaround I must use 
```ts
import {initialize} from '@electron/remote/dist/src/main';
```

**Comment by Dalqin** [comment]
Something goofy with the types, for sure.
I notice the types export `initialize` for `@electron/remote` which is misleading if the documentation says to import from `@electron/remote/main`.

**Comment by chetbox** [comment]
I believe this PR will also fix #60
